ansible/playbooks/jenkins/roles/nginx/files/nginx.crt

```diff
@@ -0,0 +1,25 @@
+-----BEGIN CERTIFICATE-----
+MIIEHTCCAwWgAwIBAgIJANRmTQQMw3YaMA0GCSqGSIb3DQEBCwUAMIGkMQswCQYD
+VQQGEwJCUjEVMBMGA1UECAwMU8ODwqNvIFBhdWxvMRcwFQYDVQQHDA5Ib3J0b2zD
+g8KibmRpYTEMMAoGA1UECgwDSUJNMQwwCgYDVQQLDANMVEMxJTAjBgNVBAMMHHN0
+YXJrLXJpY2thcmQubHRjLmJyLmlibS5jb20xIjAgBgkqhkiG9w0BCQEWE3Bvd2Vy
+a3ZtQGJyLmlibS5jb20wHhcNMTYwMTA1MTYzMjU5WhcNMTcwMTA0MTYzMjU5WjCB
+pDELMAkGA1UEBhMCQlIxFTATBgNVBAgMDFPDg8KjbyBQYXVsbzEXMBUGA1UEBwwO
+SG9ydG9sw4PCom5kaWExDDAKBgNVBAoMA0lCTTEMMAoGA1UECwwDTFRDMSUwIwYD
+VQQDDBxzdGFyay1yaWNrYXJkLmx0Yy5ici5pYm0uY29tMSIwIAYJKoZIhvcNAQkB
+FhNwb3dlcmt2bUBici5pYm0uY29tMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIB
+CgKCAQEAvqSFnHop6RAsp/IG1Olvty9PVlgPkBZrRrMJyCQ9upFUWksdxZZeksYV
+iQsKUEj+hJhGvM9zsdgX4kxlRj95BNezD+c6YgJ8uBgBFMR454rzOJPwR6EiIdxN
+Kvh/7DZIWkJwcd8ybnOwiSUCEgPM+HYkeeiiSWqtRwq6bqhahfKmeolGVQZxz+pw
+uzYEMEkJRxIlIT1396EMp/mNnhoDwUgRFWO7IFRfAVClePzd5QYu9A7b+wlnPPwn
+CQ3ugFbRWk+WgUOtMidCAAY7a5BBLMgpRaScKNeU/eBQjJNcmD1To16Us/wLNAoR
+fUPtM/W7TiNOowNbVpVkSF7pbiXAnQIDAQABo1AwTjAdBgNVHQ4EFgQUYr+J6bF5
+0M9aA8GI1LNjhzqAueUwHwYDVR0jBBgwFoAUYr+J6bF50M9aA8GI1LNjhzqAueUw
+DAYDVR0TBAUwAwEB/zANBgkqhkiG9w0BAQsFAAOCAQEAXFYatwSEVZup7fEYhk0b
+F/u8SsK68Iad2u6U99ba60HMMFfwk1vdEvdPW7mi4eX7fI2uEe7Aotx9nb7QoIgk
+DRiQzOv6S4eSFtuWPoLoIlXrqm26LgK5QwXflT1yRUkhRzlV4XDOqYpYA7/xNuPV
+6mq87E3Iakg7sYn24kEYmPVq/tzTWztYDDxQrQg9wIAh/uikDiMUjLHy4moa0FEr
+Pbf9DAlxkEP+af/il9gvIH5+V2FibjaV0Ee5Q/NB6T2vs1c83LwWgHAjyPh3EVv/
+PON9/+0E5eShCVF2m9pwkgzsK0SJC+G08bqbqT2kmjFOcXVTI0nzmg8faxV/2VfC
+jg==
+-----END CERTIFICATE-----
```

Comment:
I think `nginx.crt` and `nginx.key` should be generated automatically if user hasn't specified any file.  What do you think?